frontend/src/components/Listing/ListingCard.tsx

```diff
@@ -6,17 +6,54 @@ import Typography from '@mui/material/Typography';
 import { Button, CardActionArea, CardActions } from '@mui/material';
 import { capitalize } from '@mui/material/utils';
 import AddShoppingCartIcon from '@mui/icons-material/AddShoppingCart';
+import AddEditIcon from '@mui/icons-material/Edit';
 import 'src/styles/ListingCard.css'
+import {useParams} from "react-router-dom";
```

Comment:
![image](https://user-images.githubusercontent.com/72239525/226497843-92658f4c-f000-45f9-81fe-c0dae5f62901.png)

Remove unused inputs so that the build doesn't fail

Reply:
Removed imports